### PR TITLE
add PCTA 3.2

### PR DESCRIPTION
**Pull request by DennisMao**

### CODE REVIEW

File: PCTA/3.2.txt [hunk 1]
```diff
@@ -1,126 +1,125 @@
...
 22
 00:01:45,601 --> 00:01:51,660
-接下来我们后面操作都会使用MySQL可那一套来进行对天地面操作
+接下来我们后面操作都会使用MySQL Client来进行对TiDB操作
 
 23
 00:01:51,661 --> 00:01:54,060
-那么相同的我们使用买受可爱
-
+那么相同的,我们使用MySQL Client
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt:95]:
```suggestion

```

File: PCTA/3.2.txt [hunk 2]
```diff
@@ -132,31 +131,31 @@
...
 
 35
 00:02:33,420 --> 00:02:36,480
-是使用可以在U在命令来进行添加
+是使用Create User命令来进行用户添加
 
 36
 00:02:37,800 --> 00:02:38,100
-那时候
+..
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
这个时候
```

File: PCTA/3.2.txt [hunk 3]
```diff
@@ -132,31 +131,31 @@
...
 
 37
 00:02:38,101 --> 00:02:42,180
-我们来加一个买来的命的密码是第二
+我们来加一个myadmin的密码是123456
 
 38
 00:02:42,181 --> 00:02:42,840
-3456
+..
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
并且
```

File: PCTA/3.2.txt [hunk 4]
```diff
@@ -304,35 +303,35 @@ OK登陆到这个田地
 
 76
 00:05:23,670 --> 00:05:25,350
-为了和买车做金融
+为了和MySQL做兼容
 
 77
 00:05:25,351 --> 00:05:28,500
-我们在买车后数据库中放了亲爱滴地
+我们在mysql数据库中放了TiDB
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
我们在MySQL数据库中放了TiDB
```

File: PCTA/3.2.txt [hunk 5]
```diff
@@ -344,127 +343,127 @@ OK登陆到这个田地
...
 
 100
 00:06:53,310 --> 00:06:56,670
-我们可以看到有34个CIDB相关的参数
+OK，我们可以看到有34个TiDB相关的参数
 
 101
 00:06:59,100 --> 00:07:01,560
-这些个任务参数都是可以在线修改和调整
+这些row的参数都是可以在线修改和调整
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
一些 global 的参数都是可以在线修改和调整
```

File: PCTA/3.2.txt [hunk 6]
```diff
@@ -476,231 +475,231 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
 
 125
 00:08:31,141 --> 00:08:31,980
-一个是十点
+一个是10点
 
 126
 00:08:37,620 --> 00:08:41,760
-下面进入迪迪尔和迪亚梅尔的吃饭环节
+下面进入DDL和DML的演示(没听清)环节
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
下面进入DDL和DML的示范环节
```

File: PCTA/3.2.txt [hunk 7]
```diff
@@ -476,231 +475,231 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
 
 127
 00:08:41,761 --> 00:08:44,940
-首先我们来进行新建数据库的操作界面
+首先我们来进行，新建数据库的操作
 
 128
 00:08:44,941 --> 00:08:46,830
-我们今年两个数据
+这边我们新建两个数据
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
这边我们新建两个数据库
```

File: PCTA/3.2.txt [hunk 8]
```diff
@@ -476,231 +475,231 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
+这是一个两个字段的表
 
 145
 00:09:56,821 --> 00:09:57,540
 一个ID
 
 146
 00:09:57,541 --> 00:09:58,890
-他是逐渐一个
+他是主键
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
它是主键
```

File: PCTA/3.2.txt [hunk 9]
```diff
@@ -476,231 +475,231 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
 
 147
 00:09:58,891 --> 00:10:01,920
-那那是一个八千的字符串的子弹
+一个name,它是一个varchar的字符串的字段
 
 148
 00:10:05,790 --> 00:10:06,330
-今年完之后
+建立完之后
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
新建完之后
```

File: PCTA/3.2.txt [hunk 10]
```diff
@@ -476,231 +475,231 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
 
 149
 00:10:06,331 --> 00:10:08,250
-我们看一下当天的数据量的情况
+我们看一下当前的数据表的情况
 
 150
 00:10:09,420 --> 00:10:11,850
-额这个数据库里面你有账期的数据表
+这个数据库里面已有张t1的数据表
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
这个数据库里面已有一张t1的数据表
```

File: PCTA/3.2.txt [hunk 11]
```diff
@@ -476,231 +475,231 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
 
 151
 00:10:12,900 --> 00:10:14,550
-我们像这张表中插入三条数据
+我们向这张表中插入三条数据
 
 152
 00:10:21,060 --> 00:10:24,840
-好的我们这边使用了标准的颜色的语句来写入数据因色彩
+好的，我们这边使用了标准的Insert的语句来写入数据
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt:610]:
```suggestion
好的，我们这边使用了标准的 INSERT 的语句来写入数据
```

File: PCTA/3.2.txt [hunk 12]
```diff
@@ -476,231 +475,231 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
 
 153
 00:10:24,841 --> 00:10:27,840
-因此T指定字段名称该六
+Insert into t1指定字段名称，values指定字段值的方式
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt:614]:
```suggestion
INSERT INTO T1 指定字段名称，VALUES 指定字段值的方式
```

File: PCTA/3.2.txt (hunk 11 above)
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
好的，我们这边使用了标准的insert的语句来写入数据
```

File: PCTA/3.2.txt (hunk 12 above)
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
insert into t1 指定字段名称，values 指定字段值的方式
```

File: PCTA/3.2.txt [hunk 13]
```diff
@@ -748,139 +747,139 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
 
 189
 00:12:44,430 --> 00:12:45,870
-最后我们来只要不在吧
+最后，我们来drop table
 
 190
 00:12:56,850 --> 00:12:57,840
-我朋友变成了一个空
+已经变成一个空库
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
这个库已经变成一个空库
```

File: PCTA/3.2.txt [hunk 14]
```diff
@@ -748,139 +747,139 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
 
 198
 00:13:29,251 --> 00:13:31,320
-这个事物是需要先是打开的
+这个事务是需要显式打开的
 
 199
 00:13:32,340 --> 00:13:34,110
-这个时候我们形容一个
+这个时候我们需要一个
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
这个时候我们写入一个
```

File: PCTA/3.2.txt [hunk 15]
```diff
@@ -748,139 +747,139 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
 
 216
 00:14:38,521 --> 00:14:41,340
-让我们再进一个提交的隔离级别的偃师
+然后我们再进一个提交的，隔离级别的演示
 
 217
 00:14:46,080 --> 00:14:48,690
-我们需要在三线一当中显示的是
+我们需要在session当中显式的声明
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
同样，我们需要在 session1 当中显式的声明
```

File: PCTA/3.2.txt [hunk 16]
```diff
@@ -748,139 +747,139 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
 
 218
 00:14:48,691 --> 00:14:50,190
-你这个事物开始
+这个事务开始
 
 219
 00:14:51,300 --> 00:14:51,600
-哎
+..
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
begin
```

File: PCTA/3.2.txt [hunk 17]
```diff
@@ -892,403 +891,403 @@ OK没有做
 
 223
 00:15:01,441 --> 00:15:02,820
-然后私下数据
+4条数据
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
表中的数据，4 条数据
```

File: PCTA/3.2.txt [hunk 18]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 224
 00:15:07,470 --> 00:15:11,370
-那这个时候我们可以看一下这表出去
+这个时候我们可以看一下这个表的数据
 
 225
 00:15:15,480 --> 00:15:15,840
-29
+还是只有3号
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt:902]:
```suggestion
还是只有3行
```

File: PCTA/3.2.txt [hunk 19]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 231
 00:15:34,561 --> 00:15:35,910
-就是我们在其他的进行当中
+这时候我们在其他的session当中
 
 232
 00:15:35,911 --> 00:15:38,940
-也是能查出这个123次没有问题
+也是能查出这个1234,没有问题
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
也是能查出这个1,2,3,4,没有问题
```

File: PCTA/3.2.txt [hunk 20]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 234
 00:15:45,210 --> 00:15:48,870
-下面我们介绍一下家里面支持的数据类型
+下面我们介绍一下TiDB支持的数据类型
 
 235
 00:15:48,871 --> 00:15:49,770
-包括ID
+包括id，int类型
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
包括 id 是 int 类型
```

File: PCTA/3.2.txt [hunk 21]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 236
 00:15:51,210 --> 00:15:55,320
-印着类型纳米是打包类型什么类型
+run1 double类型，decimal类型
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt:946]:
```suggestion
num1 是 double类型，num2 是 decimal类型
```

Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
num1 是 double类型， decimal类型
```

File: PCTA/3.2.txt [hunk 22]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 244
 00:16:26,340 --> 00:16:26,760
-根据表
+分区表
 
 245
 00:16:26,761 --> 00:16:28,920
-再买车或三点也是一个比较完整的功能
+在MySQL上也是一个比较完整的功能
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
在 TiDB 3.0 上也是一个比较完整的功能
```

File: PCTA/3.2.txt [hunk 23]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 247
 00:16:31,710 --> 00:16:34,890
-我们现在实现了麦积分区和哈希分区
+我们现在实现了range分区和hash分区
 
 248
 00:16:34,891 --> 00:16:37,230
-首先我们会来新建一个睿智分区的表
+首先我们会来新建一个range分区的表
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt:994]:
```suggestion

```

File: PCTA/3.2.txt [hunk 24]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 250
 00:16:44,220 --> 00:16:46,440
-那我们就开始新建一张锐积分区的表
+那我们就开始新建一张range分区的表
 
 251
 00:16:51,570 --> 00:16:52,650
-没法预埋深位置
+(没听清)
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
语法与 MySQL 一致
```

File: PCTA/3.2.txt [hunk 25]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 252
 00:17:22,800 --> 00:17:27,630
-因为等于把我们可以看到是需要上面的字段
+这边我们可以看到是需要上面的字段
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
这边的语法我们可以看到是使用上面的字段
```

File: PCTA/3.2.txt [hunk 26]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
+下面是PARTITION BY RANGE (TO_DAYS)
 
 255
 00:17:32,671 --> 00:17:33,660
 然后这个方式
 
 256
 00:17:33,661 --> 00:17:43,530
-我们见面是把还要带着这个子弹使用初代死来精准到每一天来进行一个安天的分居的一个情况
+我们这边是把.带着这个hiredate字段使用TO_DAYS来精准到每一天来进行一个按天的分区的一个情况
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
我们这边是把带着这个hiredate字段使用TO_DAYS()来精准到每一天来进行一个按天的分区的一个情况
```

File: PCTA/3.2.txt [hunk 27]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 253
 00:17:27,631 --> 00:17:29,250
-居民是跟普通的表一样
+声明是跟普通的表一样
 
 254
 00:17:29,251 --> 00:17:32,670
-下面是有爬梯姓白瑞秋的意思
+下面是PARTITION BY RANGE (TO_DAYS)
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
下面是PARTITION BY RANGE (TO_DAYS(hiredate))
```

File: PCTA/3.2.txt [hunk 28]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 263
 00:18:25,981 --> 00:18:27,390
-这个是我们要爬T型牌
+这个时候我们用PARTITION BY HASH(id)
 
 264
 00:18:27,391 --> 00:18:29,310
-哈希ID来分成了四个
+来分成了四个分区
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
来分成了四个 hash 分区
```

File: PCTA/3.2.txt [hunk 29]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 282
 00:19:23,311 --> 00:19:25,260
-为默认打开了一个快速收集
+会默认打开了一个快速收集统计的参数
 
 283
 00:19:25,261 --> 00:19:28,950
-同期的参数就是默认调度冲压的一个老师来说
+就是默认调度抽烟的一个方式来收集统计信息
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
就是默认调度抽样的一个方式来收集统计信息
```

File: PCTA/3.2.txt [hunk 30]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 304
 00:20:17,911 --> 00:20:18,750
-删除于查查
+与查看
 
 305
 00:20:20,700 --> 00:20:22,200
-我们可以看一下第一张表的
+我们可以看一下第一张表的索引
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
同样的，我们可以看一下第一张表的索引
```

File: PCTA/3.2.txt [hunk 31]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 306
 00:20:22,201 --> 00:20:25,920
-所以我们现在看到这张表只有一个主键索引
+我们现在看到这张表只有一个主键索引
 
 307
 00:20:25,921 --> 00:20:30,630
-就是我们使用标准的创建索引的语法规则应该是
+就是我们使用标准的创建索引的语法create index
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
这个时候我们使用标准的创建索引的语法create index
```

File: PCTA/3.2.txt [hunk 32]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 310
 00:20:39,750 --> 00:20:41,910
-阿妹看T表已经有一个
+我们可以看到t1表已经有一个索引
 
 311
 00:20:43,230 --> 00:20:45,570
-所以他已经是那个自杀的一个缩影
+他已经是name字段的一个索引
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
它已经是name字段上的一个索引
```

File: PCTA/3.2.txt [hunk 33]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 315
 00:21:09,330 --> 00:21:10,770
-相当于法这边我们需要
+删除语法,这边我们使用
 
 316
 00:21:14,310 --> 00:21:16,320
-焦培娜奥米利来进行一个
+drop index来进行一个索引删除
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
drop index on 命令来进行一个索引删除
```

File: PCTA/3.2.txt [hunk 34]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
 
 319
 00:21:26,161 --> 00:21:26,550
-所以
+...
 
 320
 00:21:28,560 --> 00:21:32,370
-好我们现在可以看到这张表的这个应该是能用这个名字的
+好我们现在可以看到这张表的这个index
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
好，我们现在可以看到这张表的这个index
```

File: PCTA/3.2.txt [hunk 35]
```diff
@@ -344,127 +343,127 @@ OK登陆到这个田地
...
 
 108
 00:07:29,130 --> 00:07:30,600
-这是我的命令命令的一部分
+这是admin命令的一部分
 
 109
 00:07:32,280 --> 00:07:35,880
-没想到这边有一些最近的十个
+我们可以看到这边有一些，最近的十个
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
我们可以看到这边有一些，最近的10个
```

File: PCTA/3.2.txt [hunk 36]
```diff
@@ -476,231 +475,231 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
 
 134
 00:08:59,460 --> 00:09:01,080
-今天买375001七年
+这边mytest1已经新建
 
 135
 00:09:01,081 --> 00:09:05,250
-所以我们直接输入消灾的累死就给你查了一下
+所以我们直接输入show database这个命令
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
所以我们直接输入show databases这个命令
```

File: PCTA/3.2.txt [hunk 37]
```diff
@@ -476,231 +475,231 @@ B开头的其实是加迪B为嘛时候做金融娃放到里面的一些参数
...
 
 173
 00:11:54,240 --> 00:11:55,080
-二分时操作
+replace操作
 
 174
 00:11:55,081 --> 00:11:56,880
-在他碰到幽灵卡
+在他碰到unique key
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
在它碰到unique key
```

File: PCTA/3.2.txt [hunk 38]
```diff
@@ -892,403 +891,403 @@ OK没有做
...
+..
 
 259
 00:17:58,531 --> 00:18:00,690
 是这么一个情况
 
 260
 00:18:00,691 --> 00:18:04,470
-今晚球队参数转化成了具体的实践日记
+这边TO_DAYS函数转化成了具体的时间日期
```
Comment (zhenjiaogao) [inline PCTA/3.2.txt]:
```suggestion
这边TO_DAYS()函数转化成了具体的时间日期
```

**Review by zhenjiaogao — APPROVED** [review]

**Review by Soline324 — APPROVED** [review]
LGTM

**Comment by Soline324** [comment]
The suggestions have been committed, please take a look. @zhenjiaogao

**Review by zhenjiaogao — APPROVED** [review]
LGTM

**Pull request merged**